apps.json

```diff
@@ -1,4 +1,12 @@
 [
+{
+  "name": "NUS Buses",
+  "description": "Download the NUS Buses app to get real up to date bus arrival timings, bus routes and easily favourite the bus stops you prefer. It comes with a modern UI that is responsive and intuitive. Still in active development(2018)",
+  "author": "Marcus Koh Chi Hao",
+  "url": "https://goo.gl/jSDnQM",
```

Comment:
Are you sure you only want to link to your iOS app? You could link to a website that has links to both your Android and iOS versions, but if you don't have one at the moment you can always submit another PR in the future :)

Reply:
yeah ios currently. dont have a website to do that yet.